content/events/gcc2024/training/backend-dev/index.md

```diff
@@ -0,0 +1,28 @@
+---
+autotoc: true
+---
+
+<slot name="/events/gcc2024/header" />
+<div class="text-center">
+
+# Galaxy Backend Development
+
+</div>
+
+A quick walkthrough of the Galaxy backend geared for developers wishing to extend Galaxy's functionality. We will cover the basics of the data model layer for interacting with Galaxy's database and adding or extending API routes. Additional potential topics time permitting include but are not limited to Galaxy ObjectStores and how Galaxy runs jobs.
+
+## Meet the Instructors
+
+### John Chilton
+Penn State University <br>
+Galaxy Developer
+
+### David Lopez
+University of Freiburg <br>
+I'm David López, a Software Developer working as part of the European Galaxy Team at the University of Freiburg. Over the past three years, I've been involved full-time in the development and maintenance of the Galaxy code. This experience has provided me with in-depth insights into the intricacies of Galaxy's functionality.
+
+With this knowledge, my goal is to empower each of you to contribute effectively to the ongoing evolution of the Galaxy project. Together, we can work towards improving a platform that computational research available for everyone.
```

Comment:
I guess the Bio from the Trainers Form was completely out of place as I can see :laughing: 
```suggestion
Galaxy Developer
```